React/Base/RCTDisplayLink.m

```diff
@@ -16,6 +16,11 @@
 #import "RCTModuleData.h"
 #import "RCTProfile.h"
 
+#if TARGET_OS_OSX // TODO(macOS, https://github.com/microsoft/react-native-macos/issues/533)
+// To compile in Xcode 12 beta 4 on macOS, we need to explicitly pull in the framework to get the definition for CACurrentMediaTime()
```

Comment:
Hopefully a dumb question, but where is CACurrentMediaTime actually used? I don't see it in this compilation unit?

Reply:
Expanded from the logging macro on what is now line 115- `RCT_PROFILE_BEGIN_EVENT`